predict.py

```diff
@@ -166,12 +167,65 @@ def base_setup(
             shared_models=shared_models,
         )
 
```

Comment:
nit - since these flags are just simple little flags we set `setup` for dev/schnell predictor, I don't mind adding a separate `compile_ae` flag